physical/physical_test.go

```diff
@@ -283,7 +283,7 @@ func testBackend_ListPrefix(t *testing.T, b Backend) {
 
 func testHABackend(t *testing.T, b HABackend, b2 HABackend) {
 	// Get the lock
-	lock, err := b.LockWith("foo", "bar")
+	lock, err := b.LockWith("/core/lock/foo", "bar")
```

Comment:
What's the reason for this change? You said something about the backend not being cleaned up from the previous test, but I didn't quite follow the issue.

By far the *nicest* thing would be to actually do what you can see the Consul code doing (and some other backend code not in physical) and use `dockertest`. This will allow all of the etcd code to be tested on every CI run.